apps/framework-cli/src/framework/core/infrastructure_map.rs

```diff
@@ -724,7 +724,10 @@ mod tests {
             abs_file_path: PathBuf::new(),
         };
         // Making some changes to the map
-        new_target_primitive_map.datamodels.add(new_data_model);
+        new_target_primitive_map
+            .datamodels
+            .add(new_data_model)
+            .unwrap();
```

Comment:
Why do you need to add the `unwrap` here, is it safe?

Reply:
it's a(n ignored) test